### DISCUSSION TITLE
Find out what the SLIM build command took out of my image

### DISCUSSION THREAD

**Discussion by ConnorRyan-8451** [slimtoolkit/slim#479] [Q&A]
Hello, I am pretty new to SLIM and am wondering if there is a flag I can set to see exactly what the slim build command took out of my original image. This would be very beneficial to figuring out what more we can slim down with our alpine images.

**Comment by kcq** [comment]
> Hello, I am pretty new to SLIM and am wondering if there is a flag I can set to see exactly what the slim build command took out of my original image. This would be very beneficial to figuring out what more we can slim down with our alpine images.

It's possible to do it with Slim, but it's a bit messy. There's no built-in command for it. You can xray your original image and then xray the minified image and then you can compare the report files.

The easy way to accomplish what you want is to use the Slim SaaS where there's a nice and visual diff function: https://portal.slim.dev/login?invitecode=invite.1s85zlfnYX0p5TT1XKja49pAHbL
